src/models/gridOption.interface.ts

```diff
@@ -269,6 +269,9 @@ export interface GridOption<C extends BaseColumn = BaseColumn> {
   /** Defaults to true, which leads to render a separate span for the number and styles it with css class <i>slick-sort-indicator-numbered</i> */
   sortColNumberInSeparateSpan?: boolean;
 
+  /** Defaults to undefined. If we are inside a shadow DOM tree, this must be the shadow root of the tree */
+  shadowRoot?: ShadowRoot | undefined;
```

Comment:
you can remove the `| undefined` since `?.` already covers that

Reply:
oh true, done